### PR TITLE
fix: fix cdr gui alert toggles

### LINKED ISSUE #7178: CDR gui summary screen submit breaks when set no alert types
CDR gui summary screen submit breaks when set no alert types

Reported by @hanksterr7 at https://github.com/openemr/openemr/pull/7160#discussion_r1455479451

### PR DESCRIPTION
**Pull request by bradymiller**
fixes #7178

### CODE REVIEW

**Comment by bradymiller** [comment]
testing well

File: interface/super/rules/library/RuleManager.php [hunk 1]
```diff
@@ -498,6 +498,9 @@ function updateSummary($ruleId, $types, $title, $developer, $funding, $release,
     {
         $rule = $this->getRule($ruleId);
 
+        // if $types is empty, then set to empty array so the in_array() calls below will work
+        $types = !empty($types) ? $types : [];
```
Comment (bradymiller) [inline interface/super/rules/library/RuleManager.php:502]: actually, i think i can just use the elvis operator

**Review by hanksterr7 — COMMENTED** [review]
Confirming that with this change, the error I originally observed is resolved.

**Comment by bradymiller** [comment]
I just brought this fix into rel-702

**Pull request merged**